wmfdata/utils.py

```diff
@@ -140,7 +128,7 @@ def check_kerberos_auth():
         )
     elif klist != 0:
         raise OSError(
-          "There was an unknown issue checking your Kerberos credentials."
+            "There was an unknown issue checking your Kerberos credentials."
```

Comment:
Ditto. Space changes look unrelated to contribution?

If we need to reformat, let's do it on a separate PR.

Reply:
Ah ok, I didn't realize that they'd specifically need their own PR. Figured I'd just send them as indent rainbow was showing me that they were wrong. I guess these can be reverted then.